packages/web-components/fast-element/src/controller.ts

```diff
@@ -258,10 +272,13 @@ export class Controller extends PropertyChangeNotifier {
         }
 
         const behaviors = this.behaviors;
-
         if (behaviors !== null) {
-            for (let i = 0, ii = behaviors.length; i < ii; ++i) {
-                behaviors[i].bind(element, defaultExecutionContext);
+            const keys = behaviors.keys();
```

Comment:
How does this approach compare to a normal for/of loop? Is this faster?

Reply:
I didn't actually know you could for/of a Map but of course you can. It appears to be slightly slower in general: https://jsbench.me/pbkkit8zui/1 - I Changed this to use for/of because it's more concise and faster. Thanks for calling this out!